test/redis-sentinel/testStatesRedis.js

```diff
@@ -4,27 +4,28 @@
 /* jshint expr:true */
 'use strict';
 
-const expect  = require('chai').expect;
-const setup   = require(__dirname + '/../lib/setup4controller');
 const fs      = require('fs');
+const path    = require('path');
+const expect  = require('chai').expect;
+const setup   = require(path.join(__dirname,'..', 'lib', 'setup4controller'));
```

Comment:
you didn't have to split it all up, but fine with me :)

Reply:
if then correct ;-)